content/en/docs/contribute/style/style-guide.md

```diff
@@ -110,8 +110,12 @@ document, use the backtick (`).
   <tr><th>Do</th><th>Don't</th></tr>
   <tr><td>The <code>kubectl run</code> command creates a Deployment.</td><td>The "kubectl run" command creates a Deployment.</td></tr>
   <tr><td>For declarative management, use <code>kubectl apply</code>.</td><td>For declarative management, use "kubectl apply".</td></tr>
+  <tr><td>For code fencing use <code>```</code>.</td><td>For code fencing use <code>" "</code>.</td></tr>
 </table>
 
+{{< note >}} Syntax highlighting is supported but optional {{< /note >}}
```

Comment:
Make sure to end the sentence with a period. It's also helpful to specify what this is related to.
```
{{< note >}}
The website supports syntax highlighting for code samples, but specifying a language isn't required. For example, you can enable highlighting for Python samples by enclosing the sample like so: 
<code>```py
import foo
```</code>.
{{< /note >}}
```

Reply:
feedback taken, thanks.

Just a note that when i tested locally i was able to get highlighted code only with the below snippet

* code is highlighted and you get the nice color ;)
```
<code>
```py
import foo 
```
</code>
```

* code is not highlighted and you don't get the nice color ;)
```
<code>```py
import foo
```</code>.
```

So what am i missing here ?